server/meta.jsx

```diff
@@ -15,6 +15,9 @@ type SDKMeta = {|
     getSDKLoader : (options? : {| baseURL? : string, nonce? : string |}) => string
 |};
 
+const emailRegex = /^.+@.+$/;
```

Comment:
Should we use a more robust email regex? Or are we just shooting for a rough match?

Reply:
whoops sorry just saw this was closed 😆. disregard